### DISCUSSION TITLE
JavaAgent Benchmark testing

### DISCUSSION THREAD

**Discussion by sparam** [open-telemetry/opentelemetry-java#2570] [Q&A]
Hi,

I am trying to figure out a way to do benchmark testing the Otel Java agent and load test the OTel Collector.

There seems to be some good support for benchmark testing. Could these be used for Otel Java agent benchmark testing? Could these be executed independent of building the entire project? 
https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/master/benchmark 

Appreciate if you could point to some recommended benchmark testing.

Thanks!

**Comment by jkwatson** [comment]
Hi @sparam . I think this belongs over in the instrumentation repository, rather than here.

**Reply by sparam** [reply]
Hi @jkwatson moved to [instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation/discussions/2104). Thanks!
